packages/devtools_app/lib/src/inspector/diagnostics_node.dart

```diff
@@ -90,6 +90,12 @@ class RemoteDiagnosticsNode extends DiagnosticableTree {
   /// JSON describing the diagnostic node.
   final Map<String, Object> json;
 
+  // Describing whether the Node is an ErrorWidget
+  bool get isError => json['error'] == true;
+
+  // TODO(albertusangga): Comment
+  int get distanceFromError  => json['distanceFromError'];
```

Comment:
Here is another idea to make the multiple errors case understandable to users:
We could make the InspectorService keep track of the concept of the "selected error" just like it tracks the concept of the selected widget.
Then we can highlight the selected error in red and other errors in orange.
The selected error would default to the first error returned but users could click around in the error logging view to select a different error or deselect all errors.